src/tags.rs

```diff
@@ -9,39 +9,43 @@ use diesel::prelude::*;
 
 /// Remove a key value pair from the tags.  
 pub fn delete(args: Args) -> Result<()> {
-    let conn = DB.get()?;
-    let key = args
-        .params
-        .get("key")
-        .ok_or("Unable to retrieve param: key")?;
-
-    match diesel::delete(tags::table.filter(tags::key.eq(key))).execute(&conn) {
-        Ok(_) => args.msg.react(args.cx, "✅")?,
-        Err(_) => args.msg.react(args.cx, "❌")?,
+    if api::is_wg_and_teams(&args)? {
+        let conn = DB.get()?;
+        let key = args
+            .params
+            .get("key")
+            .ok_or("Unable to retrieve param: key")?;
+
+        match diesel::delete(tags::table.filter(tags::key.eq(key))).execute(&conn) {
+            Ok(_) => args.msg.react(args.cx, "✅")?,
+            Err(_) => args.msg.react(args.cx, "❌")?,
+        }
     }
```

Comment:
I would return an error message here. Users should be encouraged to talk to someone to add a tag.

Reply:
Currently the error is `An error occurred`, do you have a specific message in mind?

Reply:
Err... thought this was create, my bad. For delete, that's fine, but for create, it should be an explicit "talk to a Rust team/wg member" message.